index.html

```diff
@@ -1,14 +1,97 @@
+    <div id="topMenu">
+      <img id="menuButton" src="assets/hambMenu.png">
+      <a class="logoHeader" href="./index.html"><img id="logo" src="assets/logo.svg"></a>
+      <nav>
+        <a class="navItems navAbout" href="./about.html">About</a></li>
+        <a class="navItems" href="#mainProgram">Program</a></li>
+        <a class="navItems" href="#join">Join</a></li>
+        <a class="navItems" href="#partner">Sponsor</a></li>
+        <a class="navItems" href="#news">News</a></li>
+        <a class="navItems campaign" href="#CC campaign">CC campaign</a></li>
+      </nav>
+      <ul id="menu">
+        <li id="closeMenuBtn"><a class="closebtn close_menu">&times;</a></li>
+        <li class="menuItems"><a href="./about.html">About</a></li>
+        <li class="menuItems"><a href="#program">Program</a></li>
+        <li class="menuItems"><a href="#join">Join</a></li>
+        <li class="menuItems"><a href="#sponsor">Sponsor</a></li>
+        <li class="menuItems"><a href="#news">News</a></li>
+        <li class="menuItems"><a href="#CC campaign">#CC campaign</a></li>
+      </ul>
+    </div>
+  </header>
+
+  <section id="blurSection">
+  
+    <section id="home">
+      <h2>"Hello! Firefighters!"</h2>
+      <h1>FIREMEN MEETING<br>GLOBAL SUMMIT<br>2023</h1>
+      <p class="brief">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi finibus dui in eros fringilla rhoncus. Proin tempor id libero eleifend gravida.</p>
+      <p class="date">2023.10.25(WED) ~ 26(THU)</p>
+      <p class="place">@ Jaracanda Club, Quito.</p>
+    </section>
+
+    <section id="mainProgram">
+      <h3>Main Program</h3>
+      <hr>
+      <div id="programContainer"></div>
+      <div class="joinBtnHolder"><p class="join">Join FM Global Summit 2023</p></div>
+    </section><section id="aboutMe"></section>
+
+    <section id="speakers">
+      <h3>Featured Speakers</h3>
+      <hr class="speakersLine">
+      <div id="speakersContainer"></div>
+      <div id="moreSpeakersContainer"></div>
+      <div class="moreContainer">
+        <button class="moreSpeakerBtn" type="button">MORE <img src="assets/arrowMore.svg" alt="arrow" class="moreBtnArrow"></button>
+      </div>
+    </section>
+
+    <section id="partner">
+      <h2 class="sectionTitle">Partner</h2>
+      <img src="assets/about/logoSectionLine.svg" alt="line">
+      <div class="partnerContainer">
+        <img class="partnerLogo" src="assets/about/partner1.png">
+        <img class="partnerLogo" src="assets/about/partner2.png">
+        <img class="partnerLogo" src="assets/about/partner3.png">
+        <img class="partnerLogo" src="assets/about/partner4.svg">
+        <img class="partnerLogo" src="assets/about/partner5.svg">
+      </div>
+    </section>
+
+  </section>
+
+  <footer>
+    <div>
+      <img src="assets/about/ccFooter.svg">
+      <p>Firemen Meeting<br>Ecuador</p>
+    </div>
+    <p class="rightsCourse">2023 Fireman Meeting, Ecuador<br>Some Rights Reserved</p>
+  </footer>
```

Comment:
- [ ] Please 🙏 make sure your footer looks identical to the footer in the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) by increasing the font, padding on the section and adjusting the alignment.
 
Your design:

<img width="1510" alt="Screenshot 2023-03-17 at 9 39 08 PM" src="https://user-images.githubusercontent.com/44798044/225991314-b234d96e-e905-447d-8403-f8fd6bb74ac8.png">

Behance design:

<img width="1034" alt="Screenshot 2023-03-17 at 11 11 57 PM" src="https://user-images.githubusercontent.com/44798044/225991375-4e4a1f33-add0-4dad-a7aa-0b860b0251b6.png">